tendermint/src/public_key.rs

```diff
@@ -21,6 +22,9 @@ use tendermint_proto::{
 
 use crate::{error::Error, prelude::*, signature::Signature};
 
+#[cfg(feature = "secp256k1")]
+use signature::Verifier as _;
```

Comment:
So with the "secp256k1" feature on, the trait's `verify` method is brought in scope, but our `VerificationKey` has the inherent `verify` method and it wins without ambiguity. Dodgy, but it works.